api/osb/check_platform_id_plugin.go

```diff
@@ -59,6 +59,16 @@ func (p *checkPlatformIDPlugin) PollBinding(req *web.Request, next web.Handler)
 	return p.assertPlatformID(req, next)
 }
 
+// FetchService intercepts get service instance requests and check if the instance owner is the same as the one requesting the operation
+func (p *checkPlatformIDPlugin) FetchService(req *web.Request, next web.Handler) (*web.Response, error) {
+	return p.assertPlatformID(req, next)
```

Comment:
if the platform is global cf this will probably be insufficient

Reply:
I am not sure there is anything we can do to avoid this. Do you have any ideas?